CharacterCreator/CharacterCreator/Models/Characters.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharacterCreator.Models
+{
+    public enum Sex
+    {
+        Male,
+        Female
+    }
+
+    public enum Alignment
```

Comment:
Put in separate file called Alignment.cs. See above comment

Reply:
Fixed